include/BSMPT/utility/parser.h

```diff
@@ -115,6 +157,7 @@ class parser
     std::string argument;
     std::string description;
     std::optional<std::string> value;
+    std::string default_val = "";
```

Comment:
Why do you need a new store for the default value? I only find it used in the `print_help` function

Reply:
If we don't need it then there is no reason to keep it

Reply:
We use it in [BSMPT::parser prepare_parser()](https://github.com/phbasler/BSMPT/blob/e27b17b84f33f80561a1f5cdad2aac2cd213474f/src/prog/PotPlotter.cpp#L658).